crates/task-impls/src/transactions.rs

```diff
@@ -445,7 +445,7 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>, V: Versions> TransactionTask
 
                 let mut make_block = false;
                 if *view - *self.cur_view > 1 {
-                    error!("View changed by more than 1 going to view {:?}", view);
+                    warn!("View changed by more than 1 going to view {:?}", view);
```

Comment:
Maybe make this info? It seems to happen in a lot of cases

Reply:
Sounds good to me